public/components/layer_config/documents_config/style/label_config.tsx

```diff
@@ -240,12 +269,27 @@ export const LabelConfig = ({
                     placeholder={i18n.translate('maps.documents.labelTittlePlaceholder', {
                       defaultMessage: 'Add label',
                     })}
-                    value={selectedLayerConfig.style?.label?.tittle ?? ''}
-                    onChange={onStaticLabelTittleChange}
+                    value={selectedLayerConfig.style?.label?.tittleByFixed ?? ''}
+                    onChange={onFixedLabelTittleChange}
                     disabled={!selectedLayerConfig.style?.label?.enabled}
                     isInvalid={inValidLabelTittle}
                   />
                 )}
+                {selectedLayerConfig.style?.label?.tittleType === 'by_field' && (
+                  <EuiComboBox
```

Comment:
since we allow only one option, shall we use EuiSuperSelect?

Reply:
Ack, will try to replace if workable

Reply:
Added `singleSelection={{ asPlainText: true }}` to ComboBox component, so that we can have the same effect, also because the getFields functions are mainly used for ComboBox component, so I prefer reuse the same component with less change.
<img width="380" alt="image" src="https://user-images.githubusercontent.com/90288540/223634765-fafae1cc-9d6e-4aaa-b5e9-2f4425ffbf8e.png">